uploader/wiki_uploader.py

```diff
@@ -27,13 +27,18 @@ def upload_file(self, file_name, file_stream, description=""):
         if not description:
             description = file_name
 
-        upload_result = self.mw_client.upload(
-            file=file_stream,
-            filename=file_name,
-            description=description,
-            ignore=True,
-            comment="Uploaded with Google drive to commons.",
-        )
+        try:
+            upload_result = self.mw_client.upload(
+                file=file_stream,
+                filename=file_name,
+                description=description,
+                ignore=True,
+                comment="Uploaded with Google drive to commons.",
+            )
+        except mwclient.errors.APIError as e:
+            if e.args[0] == "fileexists-no-change":
+                return False, {}
```

Comment:
okey, one more thing. If we have an error, lets just return it regardless of the exception type. Right now, the client do not have any idea if the upload failed. A simple side alert on the frontend should be enough I guess with the right message.